articles/java/spring-framework/configuration-properties-azure-active-directory.md

```diff
@@ -14,28 +14,28 @@ ms.custom: devx-track-java, spring-cloud-azure, devx-track-extended-java
 > | spring.cloud.azure.active-directory.credential.client-id                          | Client ID to use when performing service principal authentication with Azure.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
 > | spring.cloud.azure.active-directory.credential.client-secret                      | Client secret to use when performing service principal authentication with Azure.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
-> | spring.cloud.azure.active-directory.enabled                                       | Whether to enable Microsoft Entra related auto-configuration. The default value is `false`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+> | spring.cloud.azure.active-directory.enabled                                       | Whether to enable Microsoft Entra ID related auto-configuration. The default value is `false`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
Should this be Microsoft Entra?

Reply:
After a double check, here should be Microsoft Entra ID. The logic is, if we are referencing the Azure AD product, replace it with Microsoft Entra ID. If we are referencing a Azure AD feature (like Azure Active Directory Conditional Access), change it to Microsoft Entra Conditional Access.